index.html

```diff
@@ -22,11 +22,39 @@
                   <li><a href="">Contact</a></li>
                   <li><a href=""><img class="" src="image/bg-envlope.png" alt="menu icon"></a></li>
               </ul>
-              <ul class="menu-icons"><li><a href="#"><img class="menuimg" src="image/menu.png" alt="menu icon"></a></li></ul>
+              <ul class="menu-icons"><li><a href="#" id="mbm-icon"><img class="menuimg" src="image/menu.png" alt="menu icon"></a></li></ul>
           </nav>
       </header>
       <main class="main-cont">
-          <article class="header-section">
+
+        
+        <div class="mbmenu-hide-show" id="hide-show" > 
+        <article class="header-section-mbm" id="mbm-article">
+              <div class="header-inner-cont-mbm">
+                  <section class="inner-section">
+                      <div class="cncel-cont">
+                          <div class="cncel">
+                              <a href="#" class="mbmenu-li-a"><img src="image/close-Icon.png" id="button" alt="close icon"></a>
+                          </div>
+                      </div>
+                      <div class="menu-list-cont">
+                          <div class="mblist-cont">
+                              <ul class="mbmenu-ul">
+                                  <li mbm-list-item><a href="#" class="mbmenu-li-a"><span class="li-span">Portfolio</span> <span><img src="image/arrow-right.png" alt="riht arrow"></span></a></li>
+                                  <li><a href="#head-section" class="mbmenu-li-a"><span class="li-span">About</span><span><img src="image/arrow-right.png" alt="riht arrow"></span></a></li>
```

Comment:
[OPTIONAL] I would suggest you change your About section to the Works section. Because when I try to press the About button it does the same thing as the Portfolio.